extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -40,6 +40,9 @@ class AmpLightbox extends AMP.BaseElement {
 
     /**  @private {?function(this:AmpLightbox, Event)}*/
     this.boundCloseOnEscape_ = null;
+
+    /** @private {boolean} */
+    this.isScrollable_ = false;
```

Comment:
It's difficult to auto detect so for now the users has to specify `scrollable` attribute themselves.